ci/azure-pipelines-sqlalchemy-compatibility.yml

```diff
@@ -192,10 +192,10 @@ stages:
             displayName: "Update pip"
 
           - script: ./scripts/install_mssql_odbc_driver.sh
-            displayName: "Install mssql odbc driver"
+            displayName: "Install mssql odbc driver and mssql-tools"
 
           - script: |
-              sqlcmd -U sa -P "ReallyStrongPwd1234%^&*" -Q "CREATE DATABASE test_ci;" -o create_db_output.txt
+              /opt/mssql-tools18/bin/sqlcmd -U sa -P "ReallyStrongPwd1234%^&*" -Q "CREATE DATABASE test_ci;" -C
```

Comment:
I am providing the full path to sqlcmd since we couldn't find it.
I also added the `-C` flag which skips certificate validation since we were failing here. Since we are connecting for tests to create a db, this doesn't present a risk.